pkg/client/cmd/install.go

```diff
@@ -44,9 +44,12 @@ func newCmdInstall(rootCmdOptions *RootCmdOptions) *cobra.Command {
 	cmd.Flags().StringVar(&options.registry, "registry", "", "A Docker registry that can be used to publish images")
 	cmd.Flags().StringVar(&options.organization, "organization", "", "A organization on the Docker registry that can be used to publish images")
 	cmd.Flags().StringVar(&options.pushSecret, "push-secret", "", "A secret used to push images to the Docker registry")
-	cmd.ParseFlags(os.Args)
+	err := cmd.ParseFlags(os.Args)
```

Comment:
@zregvart I think the build problem comes from here: `ParseFlags` is not needed here

Reply:
Should I just remove it? Lemme see...

Reply:
yes